platform/src/main/java/org/hillview/sketches/MinKRows.java

```diff
@@ -0,0 +1,69 @@
+package org.hillview.sketches;
+
+import it.unimi.dsi.fastutil.longs.Long2IntRBTreeMap;
+import it.unimi.dsi.fastutil.longs.Long2ObjectRBTreeMap;
+import org.hillview.table.api.IColumn;
+
+/**
+ * A data structure that computes the minHash of table. It stores the row indices of the k rows that
```

Comment:
In general you want to describe what the data structure does and not how it does it This allows you to easily change your mind with respect to the implementation. Can you write this comment to be more abstract?